MDPSplitView.m

```diff
@@ -45,20 +45,24 @@ static NSInteger MDPKeyToIndex(NSString *key)
     return [formatter numberFromString:[key substringFromIndex:MDPKeyPrefix.length]].integerValue;
 }
 
-@implementation MDPSplitView
+@interface MDPSplitView ()
 
-#pragma mark - NSView
+/*!
+ The number of active animations for each divider by index.
+ */
+@property (strong, nonatomic) NSCountedSet *mdp_animationCounts;
 
-+ (id)defaultAnimationForKey:(NSString *)key
+@end
+
+static void CommonInit(MDPSplitView *self)
```

Comment:
I’m a big fan of `static MDPSplitView *CommonInit(MDPSplitView *self)` since it makes `init` methods just `return CommonInit(self)`.